test/e2e/serial/workload_placement_test.go

```diff
@@ -429,6 +429,124 @@ var _ = Describe("[serial][disruptive][scheduler] workload placement", func() {
 			}, time.Minute, time.Second*5).Should(BeTrue(), "resources not restored on %q", updatedPod.Spec.NodeName)
 		})
 	})
+	Context("cluster with node/s having at least two numa zones", func() {
+		var nrtCandidates []nrtv1alpha1.NodeResourceTopology
+
+		BeforeEach(func() {
+			const requiredNUMAZones = 2
+			By(fmt.Sprintf("filtering available nodes with at least %d NUMA zones", requiredNUMAZones))
+			nrtCandidates := e2enrt.FilterZoneCountEqual(nrts, requiredNUMAZones)
+
+			const neededNodes = 1
+			if len(nrtCandidates) < neededNodes {
+				Skip(fmt.Sprintf("not enough nodes with at least %d NUMA Zones: found %d, needed %d", requiredNUMAZones, len(nrtCandidates), neededNodes))
+			}
+		})
+
+		Context("there are enough resources on one node but not in any numa zone", func() {
+			var reqResources corev1.ResourceList
+			var targetNodeName string
+			var targetNodeNRTInitial *nrtv1alpha1.NodeResourceTopology
+			BeforeEach(func() {
+				nrtCandidateNames := e2enrt.AccumulateNames(nrtCandidates)
+
+				var ok bool
+				targetNodeName, ok = nrtCandidateNames.PopAny()
+				Expect(ok).To(BeTrue(), "cannot select a node among %#v", nrtCandidateNames.List())
+				By(fmt.Sprintf("selecting node to schedule the pod: %q", targetNodeName))
+
+				var err error
+				targetNodeNRTInitial, err = e2enrt.FindFromList(nrtCandidates, targetNodeName)
+				Expect(err).NotTo(HaveOccurred())
+
+				// As we want to schedule a pod that fits into a node
+				// but not into any of its numa zones
+				// we get the target node available resources
+				// and use them as Request for the pod
+				targetNode := &corev1.Node{}
+				targetNodeKey := client.ObjectKey{Name: targetNodeName}
+				err = fxt.Client.Get(context.TODO(), targetNodeKey, targetNode)
+				Expect(err).NotTo(HaveOccurred())
+				reqResources = targetNode.Status.Allocatable
+
+				By("Padding all other candidate nodes")
+				var paddingPods []*corev1.Pod
+				for _, nodeName := range nrtCandidateNames.List() {
+					podName := fmt.Sprintf("padding%s-full", nodeName)
+
+					node := &corev1.Node{}
+					nodeKey := client.ObjectKey{Name: nodeName}
+					err := fxt.Client.Get(context.TODO(), nodeKey, node)
+					Expect(err).NotTo(HaveOccurred())
+
+					// calculate resources to fill node
+					podRes := node.Status.Allocatable
```

Comment:
Is having a pod that asks for the complete amount of allocatable resources is possible?

Reply:
After checking ... no, it is not. So I would have to figure out another way to be sure there is no resources on the other nodes ...

Reply:
Leaving alone the big error of assign directly one map to the other .... 
if k8s docs said "The allocatable block indicates the amount of resources on a Node that is available to be consumed by normal Pods." and there are no quotas ... using Allocatable Memory and CPU values should give a valid ResourceList for a pod , isn't it?